packages/webapp/pages/account/notifications.tsx

```diff
@@ -274,7 +274,11 @@ const AccountNotificationsPage = (): ReactElement => {
       });
     }
 
-    subscribePersonalizedDigest({ type, hour: preferredHour });
+    subscribePersonalizedDigest({
+      type,
+      hour: preferredHour,
+      sendType: personalizedDigestType as SendType,
+    });
```

Comment:
```suggestion
    subscribePersonalizedDigest({type, hour: preferredHour, sendType: personalizedDigestType as SendType});
```
Should make linter happy

Reply:
Getting this unfortunately 😢 

![Screenshot 2024-06-28 at 6 00 11 PM](https://github.com/dailydotdev/apps/assets/13744167/1adcba78-1031-4d44-ab37-a146b49fe23d)

Reply:
Ah, maybe Vercel was going off a old commit when it said it failed because of linting.